cypress/integration/legendSet.cy.js

```diff
@@ -0,0 +1,313 @@
+import { DIMENSION_ID_EVENT_DATE } from '../../src/modules/dimensionConstants.js'
+import {
+    ANALYTICS_PROGRAM,
+    TEST_DIM_LEGEND_SET,
+    TEST_REL_PE_LAST_YEAR,
+} from '../data/index.js'
+import { typeInput } from '../helpers/common.js'
+import { openDimension, selectEventProgram } from '../helpers/dimensions.js'
+import {
+    clickMenubarOptionsButton,
+    clickMenubarUpdateButton,
+} from '../helpers/menubar.js'
+import {
+    clickOptionsModalUpdateButton,
+    clickOptionsTab,
+    expectLegendDisplayStrategyToBeByDataItem,
+    expectLegendDisplayStrategyToBeFixed,
+    expectLegendDisplayStyleToBeFill,
+    expectLegendDisplayStyleToBeText,
+    expectLegendKeyOptionToBeEnabled,
+} from '../helpers/options.js'
+import { selectRelativePeriod } from '../helpers/period.js'
+import { expectRouteToBeEmpty } from '../helpers/route.js'
+import {
+    expectAOTitleToBeValue,
+    expectLegedKeyToMatchLegendSets,
+    expectLegendKeyToBeHidden,
+    expectLegendKeyToBeVisible,
+    expectTableToBeVisible,
+} from '../helpers/table.js'
+import { EXTENDED_TIMEOUT } from '../support/util.js'
+
+const event = ANALYTICS_PROGRAM
+const dimensionName = TEST_DIM_LEGEND_SET
+const periodLabel = event[DIMENSION_ID_EVENT_DATE]
+
+describe('Options - Legend', () => {
+    const defaultBackgroundColor = 'rgb(255, 255, 255)'
+    const defaultTextColor = 'rgb(33, 41, 52)'
+
+    const TEST_CELLS = [
+        { value: 60, color: 'rgb(66, 146, 198)' },
+        { value: 35, color: 'rgb(158, 202, 225)' },
+    ]
+
+    const TEST_LEGEND_SET = { name: 'Alert', color: 'rgb(237, 227, 99)' }
+
+    const assertCellsHaveDefaultColors = (selector) =>
+        cy
+            .getBySel('table-body')
```

Comment:
Is it possible to use any of the helper functions in helpers/table.js ?

Reply:
No, this is quite specific to legend sets. Also I didn't want to chain the `.each` to a proprietary function, so I thought I'd stick to vanilla for the whole line.